.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-virt32:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build 
+        run: |
+          docker run --rm -v "${PWD}:/so3" ghcr.io/andrecostaaa/so3-env:main bash -c "cd so3 && make virt32_defconfig && make -j"
```

Comment:
What about we transfer the so3-env repository to the smartobjectoriented organisation?

Reply:
We can, i always prefer separating things and since this repo already has a dockerfile i didn't want to pollute it with another one
Having said that, i also can see the benefits of having everything together

I can create a PR with the dockerfile needed and the ci to automatically build the image

Reply:
Sounds good to me. Created #73 for that